samples/README.md

```diff
@@ -16,7 +16,7 @@ docker run --rm mcr.microsoft.com/dotnet/framework/samples
 
 ## Try a pre-built ASP.NET Docker Image
 
-You can quickly run a container with a pre-built [sample ASP.NET Docker image](https://hub.docker.com/r/microsoft/dotnet-framework-samples/), based on the [ASP.NET Docker sample].
+You can quickly run a container with a pre-built [sample ASP.NET Docker image](../README.samples.md), based on the [ASP.NET Docker sample].
```

Comment:
The link to "ASP.NET Docker sample" is missing here.

Reply:
added `(aspnetapp/README.md)`